k9mail/src/main/res/values/strings.xml

```diff
@@ -1019,6 +1019,8 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="unread_widget_select_account">Show unread count for…</string>
     <string name="unread_widget_account_title">Account</string>
     <string name="unread_widget_account_summary">The account for which the unread count should be displayed</string>
+    <string name="unread_widget_unified_inbox_account_summary">Unified Inbox</string>
```

Comment:
What's the reason for not using the existing strings for "Unified Inbox" and "All messages"?

Reply:
They were in different contexts

Reply:
@cketti removed the line.